README_first.ipynb

```diff
@@ -1,60 +1,124 @@
 {
```

Comment:
<p>Ideas are the wrong way round, should be:</p><ul><li>2 Lines of code</li><li>NLP: Sentiment Analysis and NER</li><li>Wav2Vec2 inference</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>